charts/datadog/README.md.gotmpl

```diff
@@ -68,27 +72,41 @@ Now, the installation command contains the reference to the secret.
 
 ```bash
 helm install --name <RELEASE_NAME> \
-  --set datadog.apiKeyExistingSecret=$DATADOG_SECRET_NAME datadog/datadog
+  --set datadog.apiKeyExistingSecret=$DATADOG_API_SECRET_NAME datadog/datadog
 ```
 
-**Note**: Provide a secret for the application key (AppKey) using the `datadog.appKeyExistingSecret` chart variable.
-
 ### Enabling the Datadog Cluster Agent
 
+The Datadog Cluster Agent is now enabled by default.
+
 Read about the Datadog Cluster Agent in the [official documentation](https://docs.datadoghq.com/agent/kubernetes/cluster/).
 
-Run the following if you want to deploy the chart with the Datadog Cluster Agent:
+#### Custom Metrics Server
+
+If your plan to use the [Custom Metrics Server](https://docs.datadoghq.com/agent/cluster_agent/external_metrics/?tab=helm) feature, Provide a secret for the application key (AppKey) using the `datadog.appKeyExistingSecret` chart variable.
+
+```bash
+DATADOG_APP_SECRET_NAME=datadog-app-secret
+kubectl create secret generic $DATADOG_APP_SECRET_NAME --from-literal app-key="<DATADOG_APP_KEY>" --namespace="default"
```

Comment:
I don’t think we should enforce the `default` namespace.
The `helm install` commands are not specifying any namespace and I think this secret should be in the same namespace as the agent itself.

```suggestion
kubectl create secret generic $DATADOG_APP_SECRET_NAME --from-literal app-key="<DATADOG_APP_KEY>"
```